### PR TITLE
docs: Add a translation of an example in actions.md of guide docs

### PR DESCRIPTION
**Pull request by progresivoJS**
This PR updates KR translation for actions.md in guide docs.

### CODE REVIEW

**Comment by kiaking** [comment]
Thanks for the PR!

@ChangJoo-Park Could you review this PR please 🙏

File: docs/kr/guide/actions.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ export default {
   methods: {
     ...mapActions([
       'increment' // this.increment()을 this.$store.dispatch('increment')에 매핑
+
+      // mapActions는 payloads도 지원을 한다.
+      'incrementBy' // this.incrementBy(amount)를 this.$store.dispatch('incrementBy', amount)에 매핑
```
Comment (ChangJoo-Park) [inline docs/kr/guide/actions.md:116]:
```suggestion
      'incrementBy' // this.incrementBy(amount)를 this.$store.dispatch('incrementBy', amount)에 매핑
```

이상한 특수문자가 포함되어 있습니다

**Comment by ChangJoo-Park** [comment]
Thank you for ping me. @kiaking  :+1: 

I reviewed some words with suggestions.

File: docs/kr/guide/actions.md [hunk 2]
```diff
@@ -111,6 +111,9 @@ export default {
   methods: {
     ...mapActions([
       'increment' // this.increment()을 this.$store.dispatch('increment')에 매핑
+
+      // mapActions는 payloads도 지원을 한다.
```
[the remark this answers is not in the document]
Comment (progresivoJS) [reply docs/kr/guide/actions.md]: 네, 그게 더 좋아보입니다.

File: docs/kr/guide/actions.md (hunk 1 above)
Comment (progresivoJS) [reply docs/kr/guide/actions.md]: 감사합니다.

**Comment by ChangJoo-Park** [comment]
@progresivoJS 확인했습니다 감사합니다 :+1: 

@kiaking , Please merge this PR. :smile:

**Comment by kiaking** [comment]
Yay! Thanks for the checks 🙌

**Pull request merged**